ci-test.sh

```diff
@@ -31,8 +31,8 @@ else
 
     ./examples/build_all_docker.sh --check
     echo "Testing all examples"
-    ./examples/test_all.sh
-    # TODO re-enable when parsing unknown sections is fixed in twiggy 
+    ./examples/test_all.sh --no-js
```

Comment:
If we add JS support to buildkite, we can run near-runner tests there, too. I think this is probably low priority, given that I added a dedicated GitHub action to run the near-runner tests.

Reply:
Should this maybe be the opposite flag, to require enabling it to run js tests? Would be best to not require contributors to have node setup or know about this flag to disable

Reply:
I was thinking that `test_all` should run all the tests by default, but I'm ok making it run only Rust tests unless told otherwise.

Reply:
as long as the js tests still run in the CI, yeah. Can make a note in `CONTRIBUTING.md` if you'd like, doesn't really matter though

Reply:
I added help output to the `test_all.sh` script. It's right at the top of the file to make it discoverable whether people try to run the file with `-h`/`--help` or if they read the file. Do you think it's worth also adding info about it to CONTRIBUTING.md?